README.md

```diff
@@ -41,24 +41,36 @@ We have introduced this feature with [APIM CLI](https://github.com/wso2/product-
 
 ##### Deploying APIM in  K8S Cluster
 
-Kubernetes artifacts to deploy APIM and APIM analytics deployment are shipped with the this distribution.
+Kubernetes artifacts to deploy APIM and APIM analytics deployment are shipped with the this distribution. Below commands allow users to run API Manager default profile and API Manager Analytics profile in kubernetes runtime. Please note you will need to have enough menory and CPU to run below commands.
```

Comment:
Spelling mistake in 'menory'. If we are saying you need enough memory and CPU, we need to indicate how much. Users will get confused otherwise how much they need.